inventory/sample/hosts.ini

```diff
@@ -1,11 +1,11 @@
 [master]
-192.168.30.38
-192.168.30.39
-192.168.30.40
+control1 ansible_host=192.168.30.38
```

Comment:
I am wondering if it just makes sense to create another collection rather than update this collection with variables.  Is it possible to create a `test.ini`  and use that for testing only without adding these variables to `host.ini`?  Asking because 99.99% of users won't ever use the tests however now they have to discern test variables.

Reply:
It's not a blocker, just asking the question :)

Reply:
I would agree. A separate test.ini maybe in a tests directory would be good. Maybe even just put it in the .github directory.

Reply:
Good point, I have dropped that part of the patch 👍🏻

Reply:
Thank you!